validation/mesoscale_turbulence/eddying_channel_nonhydrostatic.jl

```diff
@@ -9,7 +9,7 @@ using Oceananigans.Diagnostics
 using Oceananigans.Utils
 using Oceananigans.AbstractOperations
 using Oceananigans.Advection
-using Oceananigans.TurbulenceClosures: Vertical, Horizontal
+using Oceananigans.TurbulenceClosures
```

Comment:
Can we also delete this line (I think there are more that remain)

Reply:
Yeap